vowpalwabbit/distributionally_robust.h

```diff
@@ -139,63 +134,11 @@ namespace distributionally_robust
-      if (!read) { msg << "_chisq_chi_scored " << duals.first << "\n"; }
-      bin_text_read_write_fixed(
-          model_file, reinterpret_cast<char*>(&duals.first), sizeof(duals.first), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_duals_stale " << duals_stale << "\n"; }
-      bin_text_read_write_fixed(
-          model_file, reinterpret_cast<char*>(&duals_stale), sizeof(duals_stale), read, msg, text);
-
-      duals.second.save_load(model_file, read, text);
-
-      if (!read) { msg << "_chisq_chi_alpha " << alpha << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&alpha), sizeof(alpha), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_tau " << tau << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&tau), sizeof(tau), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_wmin " << wmin << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&wmin), sizeof(wmin), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_wmax " << wmax << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&wmax), sizeof(wmax), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_rmin " << rmin << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&rmin), sizeof(rmin), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_rmax " << rmax << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&rmax), sizeof(rmax), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_n " << n << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&n), sizeof(n), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_sumw " << sumw << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&sumw), sizeof(sumw), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_sumwsq " << sumwsq << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&sumwsq), sizeof(sumwsq), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_sumwr " << sumwr << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&sumwr), sizeof(sumwr), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_sumwsqr " << sumwsqr << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&sumwsqr), sizeof(sumwsqr), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_sumwsqrsq " << sumwsqrsq << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&sumwsqrsq), sizeof(sumwsqrsq), read, msg, text);
-
-      if (!read) { msg << "_chisq_chi_delta " << delta << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&delta), sizeof(delta), read, msg, text);
-    }
-
     ScoredDual recompute_duals();
     static double chisq_onedof_isf(double alpha);
     const double& effn() { return n; }
+    friend size_t VW::model_utils::process_model_field(
```

Comment:
Same as other friend func for consistency

Reply:
Same as above